app/src/utils.js

```diff
@@ -16,3 +18,9 @@ export const isValidAddress = (address) =>
 export const getBufferToHex = (buffer) => cryptography.utils.bufferToHex(buffer);
 
 export const getTransactionBytes = (transaction) => transaction.getBytes(transaction);
+
+export const setRendererPermissions = (win) => {
+  win.browser.webContents.session.setPermissionRequestHandler((_, permission, callback) => {
```

Comment:
https://www.electronjs.org/docs/latest/tutorial/security#5-handle-session-permission-requests-from-remote-content

probably follow the electron recommendation?